AdyenPayment.php

```diff
@@ -34,6 +34,7 @@ class AdyenPayment extends Plugin
     const NAME = 'AdyenPayment';
```

Comment:
This class is getting pretty big.
Note for later we should think about refactoring new code to delegated services.